cli/smartcontract/smart_contract.go

```diff
@@ -84,6 +84,9 @@ func init() {
 func RuntimeNotify(args []any) {
     runtime.Notify(notificationName, args)
 }`
+
+	// outAutocompletionHelp is the help message used for compilation-related output flags help.
+	outAutocompletionHelp = " (if non of --out, --manifest, --config, --bindings flags are specified, then the output filenames for these flags will be guessed using the name provided via --in option by trimming/adding corresponding suffixes to the common part of the path)"
```

Comment:
none of?

Reply:
```
$ ./bin/neo-go contract compile --help
NAME:
   neo-go contract compile - compile a smart contract to a .nef file

USAGE:
   neo-go contract compile -i path [-o nef] [-v] [-d] [-m manifest] [-c yaml] [--bindings file] [--no-standards] [--no-events] [--no-permissions]

OPTIONS:
   --in value, -i value        Input file for the smart contract to be compiled
   --out value, -o value       Output of the compiled contract
   --verbose, -v               Print out additional information after a compiling
   --debug value, -d value     Emit debug info in a separate file
   --manifest value, -m value  Emit contract manifest (*.manifest.json) file into separate file using configuration input file (*.yml)
   --config value, -c value    Configuration input file (*.yml)
   --no-standards              do not check compliance with supported standards
   --no-events                 do not check emitted events with the manifest
   --no-permissions            do not check if invoked contracts are allowed in manifest
   --bindings value            output file for smart-contract bindings configuration
```
I fear this line is just too long (espcially if concatenated with others). Maybe we can explain this with `UsageText` for `contract compile`?

Reply:
> none of?

Fixed.

> Maybe we can explain this with UsageText

We use UsageText for the example of command usage, I think it's better to do it via Description for consistency with other commands. Looks good:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ ./bin/neo-go contract compile --help
NAME:
   neo-go contract compile - compile a smart contract to a .nef file

USAGE:
   neo-go contract compile -i path [-o nef] [-v] [-d] [-m manifest] [-c yaml] [--bindings file] [--no-standards] [--no-events] [--no-permissions]

DESCRIPTION:
   Compiles given smart contract to a .nef file and emits other associated
   information (manifest, bindings configuration, debug information files) if
   asked to. If non of --out, --manifest, --config, --bindings flags are specified,
   then the output filenames for these flags will be guessed using the contract
   name or path provided via --in option by trimming/adding corresponding suffixes
   to the common part of the path."

OPTIONS:
   --in value, -i value        Input file for the smart contract to be compiled (*.go file or directory)
   --out value, -o value       Output of the compiled contract
   --verbose, -v               Print out additional information after a compiling
   --debug value, -d value     Emit debug info in a separate file
   --manifest value, -m value  Emit contract manifest (*.manifest.json) file into separate file using configuration input file (*.yml)
   --config value, -c value    Configuration input file (*.yml)
   --no-standards              do not check compliance with supported standards
   --no-events                 do not check emitted events with the manifest
   --no-permissions            do not check if invoked contracts are allowed in manifest
   --bindings value            output file for smart-contract bindings configuration

```